rc/extra/tmux-repl.kak

```diff
@@ -1,6 +1,8 @@
 # http://tmux.github.io/
 # ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
 
+declare-option str repl_pane_id
```

Comment:
I'd suggest adding `-docstring` and prefix with `tmux_`, since this pane id doen't make sense for other repl mechanisms.

Reply:
This reminds me... one lesson I learned in my kakrc for using fzf-tmux, is that kak should test $kak_client_TMUX and run tmux like `TMUX=$kak_client_TMUX tmux display-message ...`.  Otherwise, if you connect to a tmux daemonized from outside the tmux, you don't have access to window commands.